console/network/src/mainnet_v0.rs

```diff
@@ -136,7 +136,6 @@ impl Network for MainnetV0 {
 
     /// A list of (consensus_version, block_height) pairs indicating when each consensus version takes effect.
     /// Documentation for what is changed at each version can be found in `ConsensusVersion`.
-    #[cfg(not(any(test, feature = "test")))]
```

Comment:
Please revert this file to what it was before this PR. The tests will break without this.